### ISSUE TITLE
FileNotFoundError on latest versions on vyper/ape/python [APE-1185]

### ISSUE THREAD

**Issue by victor-ego** [ApeWorX/ape#1538]
### Environment information

- OS: macOS 
- Python Version: 3.11
- `ape` and plugin versions: 

```sh
$ ape --version
0.6.13

$ vyper --version
0.3.9
```

### What went wrong?

Please include information like:

- what command you ran: any `ape` command
- full output of the error you received
``` 
  File "/Users/ve/miniconda3/envs/sx/lib/python3.11/site-packages/ens/utils.py", line 39, in <module>
    from ._normalization import (
  File "/Users/ve/miniconda3/envs/sx/lib/python3.11/site-packages/ens/_normalization.py", line 57, in <module>
    with open(os.path.join(specs_dir_path, "normalization_spec.json")) as spec:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/ve/miniconda3/envs/sx/lib/python3.11/site-packages/ens/specs/normalization_spec.json'
```
### How can it be fixed?
it appears that with this current setup and versions ape cant find `normalization_spec.json` when the only similar document is `_normalization.py`

**Comment by fubuloubu** [comment]
This appears like a Web3py issue, can you share a printout of `pip freeze`?

**Comment by kclowes** [comment]
We just released 6.6.1 and yanked 6.6.0 from pypi - is it working now?

**Comment by victor-ego** [comment]
updating web3py fixes the issue.
